### PR TITLE
MopDetailsView and SettingsView

### PR DESCRIPTION
**Pull request by klaudialax**
fixed repetitions in code in SettingsView.js
added displaying usage, free spaces... in MopDetailsView.js
added displaying facilities in MopDetailsView.js

### CODE REVIEW

File: src/components/MapMopsView.js [hunk 1]
```diff
@@ -107,7 +107,7 @@ export default class MapMopsView extends Component {
       >
         <Text h4 numberOfLines={2}>{marker.title}</Text>
         <Text numberOfLines={1}>Kierunek: {marker.direction}</Text>
-        {FACILITIES.getFacilitiesIcons(marker.facilities_short)}
+        {FACILITIES.getFacilitiesIconsShort(marker.facilities_short)}
```
Comment (klaudialax) [inline src/components/MapMopsView.js]: This code is not working :/ You need to remove line 169: style={m()}.
Sorry, forgot about it.

File: src/components/UsageTable.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+  createHeader = (vehicle) => {
+    return (
+      <View>
+          <Icon name={vehicle.icon} color={THEMES.basic.backgroundLightColor}/>
+          <Text style={{textAlign: 'center'}}>{vehicle.name}</Text>
+      </View>
+    )
+  }
+
+  createHeaderRow = () => {
```
Comment (klaudialax) [inline src/components/UsageTable.js]: this repetitions will be fixed in next pull request :D

File: src/components/MapMopsView.js (hunk 1 above)
Comment (klaudialax) [inline src/components/MapMopsView.js]: this code is not working :/ please, change line 169 to 
style={styles.map}
I'll push it with my next changes

**Review by oroko123 — APPROVED** [review]
1. AsyncStorage and ScrollView imports unused in UsageTable
2. Lets delete "red" from colors 
3. Maybe it is time to fix downloading MOPs, when a new list of MOPs is got from API

**Pull request merged**